core/overlays/stage/imagestreamtags.yaml

```diff
@@ -1,70 +1,69 @@
----
 apiVersion: image.openshift.io/v1
 kind: ImageStream
 metadata:
   name: thoth-toolbox
 spec:
   tags:
-    - name: latest
-      from:
-        kind: DockerImage
-        name: "quay.io/thoth-station/thoth-toolbox:v0.2.0"
-      importPolicy: {}
-      referencePolicy:
-        type: Local
+  - name: latest
```

Comment:
is the indentation correct?

Reply:
yes, it works both ways (like and with one more tab space)

Reply:
the imagestream in test get indented like this as it update by sesheta , so just followed it

Reply:
we follow :robot: